src/netlify/server.ts

```diff
@@ -1,3 +1,4 @@
+import { URI } from "@adviser/cement";
```

Comment:
we don't get to have non-platform npm dependencies in our servers, at least, doing that is a substantial change.

Reply:
for instance many of our server install paths don't do a full npm install for our dependencies, and requiring that would change the DX